docs/dev/Updates.md

```diff
@@ -0,0 +1,21 @@
+# Updates
+
+[Back to developer documentaion main page](README.md)
+
+Infomation about the QCs update process, and how we can use it should be placed here.
+
+## How it works
+
+the update process first talks to an API to see if there's anything available and then downloads the update archive right
+
+## Man In The Middle updates
+
+SSL checks are disabled everywhere (in ZenUI too, not just the updater) ... this means that with some work we would have command execution on the unit without even opening it.
```

Comment:
I'm not sure if this was the case for ZenUI, and the MITM is purely on paper. Leaving that out for now.